docs/functions/admin_guide/tracing-functions.html

```diff
@@ -148,7 +151,7 @@ <h3 class="title"><a href="../../index.html">Documentation</a></h3>
   </ul>
 </nav>
   <!--
-  <div class="edit-this-page"><a href="file:///home/abrennan/git-repos/docs-1/modules/ROOT/pages/functions/admin_guide/tracing-functions.adoc">Edit this Page</a></div>
+  <div class="edit-this-page"><a href="https://github.com/abrennan89/docs-1/edit/knfunc/modules/ROOT/pages/functions/admin_guide/tracing-functions.adoc">Edit this Page</a></div>
```

Comment:
This probably shouldn't point to your fork, should it?

Reply:
I don't think this actually shows up as anything in the site either way? https://openshift-knative.github.io/docs/docs/functions/admin_guide/tracing-functions.html

I'm not sure why this is here because I didn't add it, so I'm guessing it's some default setting in the site. I don't know that I want to spend a lot of time on it but if it's concerning you I can try to find a way to remove it?